apps/app/src/configs/fulfillment.neutron.ts

```diff
@@ -49,7 +49,8 @@ export const fulfillmentSettings: FulfillmentSettings = {
     sendToPatient: false, // disabled for org
     sendToPatientUsers: [
       // enabled for these users
-      ...(process.env.REACT_APP_TIM_USER_ID ? [process.env.REACT_APP_TIM_USER_ID] : [])
+      ...(process.env.REACT_APP_TIM_USER_ID ? [process.env.REACT_APP_TIM_USER_ID] : []),
+      ...(process.env.REACT_APP_RADO_USER_ID ? [process.env.REACT_APP_RADO_USER_ID] : [])
```

Comment:
nit: want to just put these in the array? the env var feels like overkill

Reply:
ah good call, kind of remnant stuff